packages/nodes-base/nodes/PlaceholderTrigger/PlaceholderTrigger.node.ts

```diff
@@ -0,0 +1,29 @@
+import { IExecuteFunctions } from 'n8n-core';
+import { INodeExecutionData, INodeType, INodeTypeDescription } from 'n8n-workflow';
+
+export class PlaceholderTrigger implements INodeType {
```

Comment:
why do we need a new node for this? Seems to me like it's more of a button.. it should not be selectable at all